AppRun

```diff
@@ -20,7 +20,7 @@ export PCSCLITE_CONFIG_DIR="${APPDIR}/etc/reader.conf.d"
 PCSCD_RUN_DIR="/tmp/pcscd/run"
 rm -rf "${PCSCD_RUN_DIR}"
 mkdir -p "${PCSCD_RUN_DIR}"
-echo "starting pcscd with ${APPDIR}/usr/bin/pcscd -f &"
+echo "starting pcscd with ${APPDIR}/usr/bin/pcscd --foreground --auto-exit &"
```

Comment:
Uhm... this just changed the `echo` but not the actual command below? 🤔

Reply:
oh nice good catch!